lib/file_transfer_agent/encrypt_util.js

```diff
@@ -32,6 +35,62 @@ function EncryptionMetadata(key, iv, matDesc) {
 
 exports.EncryptionMetadata = EncryptionMetadata;
 
+function TempFileGenerator() {
+
+  this.fileSync = function (option = { dir: os.tmpdir(), prefix: '', postfix: '', extension: '' }) {
+    const randomName = crypto.randomUUID();
+    const fileName = `${option.prefix || ''}${randomName}${option.postfix || ''}${option.extension || ''}`;
```

Comment:
We need dot (.) before extension when its not empty.

Reply:
Added.